project_national/housing_characteristics/Bedrooms.tsv

```diff
@@ -1,49 +1,49 @@
 Dependency=Geometry Building Type RECS	Dependency=Geometry Floor Area	Option=1	Option=2	Option=3	Option=4	Option=5	source_count	source_weight	sampling_probability
-Mobile Home	0-499	0.604737	0.395263	0	0	0	109	265141	0.00373119
-Mobile Home	1000-1499	0.020451	0.29947	0.637349	0.03716	0.005569	897	2.26131e+06	0.019923
```

Comment:
@afontani - The small perturbation in distr. values are from [ResStock-estimaton PR200](https://github.com/NREL/resstock-estimation/pull/200), where we changed the string format from `%.6f` to `%g`. In that PR, the tsv_makers were not rerun so we are seeing all the changes here.

Here's a sample line from the bedrooms.tsv when `%.6f` is turned back on. Note sampling_probability is different below because it's using the new sampling_prob calc from [ResStock-estimaton PR203](https://github.com/NREL/resstock-estimation/pull/203)
```
Mobile Home	1000-1499	0.020451	0.299470	0.637349	0.037160	0.005569	897.000000	2261309.759463	0.022442
```